### ISSUE TITLE
Specify a window object

### ISSUE THREAD

**Issue by jcbp** [LukasBombach/Lazyloader#6]
It would be really useful if I could specify a different window/document on which to load a file, for example into a contentWindow of an iframe.

**Comment by LukasBombach** [comment]
Good idea! Do you have a suggestion for a way to specify it (as a user of the lib)?

**Comment by jcbp** [comment]
Maybe it could have a method to get a new instance of LazyLoader by specifying a different window than itself.
Something like this:
`LukesLazyLoader.getInstance(iframe.contentWindow).load('your', 'css', 'and', 'js', 'files').then(callback).then(anotherCallback);`
